charts/shoot-core/components/charts/monitoring/charts/node-exporter/templates/node-exporter.yaml

```diff
@@ -107,8 +107,13 @@ spec:
             cpu: 50m
             memory: 50Mi
           limits:
-            cpu: 50m
-            memory: 50Mi
+          {{- if .Values.global.vpaEnabled }}
+            cpu: 150m
+            memory: 150Mi
+          {{- else }}
+            cpu: 200m
+            memory: 2000Mi
```

Comment:
2 GiB? This can easily brake small nodes when the node exporter try to consume too much memory.

Reply:
Oops, that was a typo. My bad.. please excuse.. I have reset it to 200Mi.